translate/docs/usage.rst

```diff
@@ -1,4 +1,4 @@
-Using the Translate Client
+Translation v2 Usage Guide
```

Comment:
Should we be renaming this file to `usage_v2.rst` / `v2_usage.rst`?  And why do we not publish usage examples for V3?

Reply:
> Should we be renaming this file to usage_v2.rst / v2_usage.rst?

We're still relying on a lot of redirects, so I'd prefer not to change the name of the file to preserve the same path. 

>  And why do we not publish usage examples for V3?

v3beta1 and v3 are generated clients, so generated snippets are in client guide and proper samples live on cloud.google.com.